File: tests/codegen/snapshots/from_graphql_schema/typescript/variables.ts
type OperationNameResult = {
    withInputs: boolean
}

type PersonInput = {
    name: string
    age: number
}

type ExampleInput = {
    id: string
    name: string
    age: number
    person: PersonInput
    people: PersonInput
    optionalPeople: PersonInput
}

type OperationNameVariables = {
    id: string | undefined
    input: ExampleInput
    ids: string[]
    ids2: (string | undefined)[] | undefined
    ids3: ((string | undefined)[] | undefined)[] | undefined
}

File: tests/codegen/snapshots/from_graphql_schema/python/variables.py
from typing import List, Optional

class OperationNameResult:
    withInputs: bool

class PersonInput:
    name: str
    age: int

class ExampleInput:
    id: str
    name: str
    age: int
    person: PersonInput
    people: PersonInput
    optionalPeople: PersonInput

class OperationNameVariables:
    id: Optional[str]
    input: ExampleInput
    ids: List[str]
    ids2: Optional[List[Optional[str]]]
    ids3: Optional[List[Optional[List[Optional[str]]]]]
